content/en/05-use-case-tutorials/network-monitoring.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Network Monitoring"
+metaTitle: "Using Pixie | Use Cases | Network Monitoring"
+metaDescription: "Network monitoring using Pixie."
+order: 1
+---
+
+This tutorial will demonstrate how to use Pixie to:
```

Comment:
Maybe add a preamble sentence here? Like Diagnosing issues at the network layer of your stack can be challenging without proper visibility into the traffic. 

"After deploying Pixie, you can easily:"

Reply:
Is this better? 

"Application health and network performance are inextricably linked. With Pixie, you can easily monitor your network alongside your applications and infrastructure.

This tutorial will demonstrate how to use Pixie to:"

Reply:
Network performance can have a big impact on the health of your services. However, problems in network performance can often be hard to detect when debugging. With Pixie, you can easily monitor your network alongside your applications and infrastructure.